packages/wonder-blocks-clickable/src/components/__docs__/clickable.argtypes.js

```diff
@@ -0,0 +1,225 @@
+export default {
+    children: {
+        description:
+            "The child of Clickable must be a function which returns the component which should be made Clickable.  The function is passed an object with three boolean properties: hovered, focused, and pressed.",
+        control: {
+            type: "text",
+        },
+        type: {
+            required: true,
+        },
+        table: {
+            type: {
+                summary: "(ClickableState) => React.Node",
+            },
+        },
+    },
+    id: {
+        description: "An optional id attribute.",
+        table: {
+            type: {
+                summary: "string",
+            },
+        },
+        control: {
+            type: "text",
+        },
+    },
+    /**
+     * States
+     */
+    light: {
+        description:
+            "Whether the Clickable is on a dark colored background. Sets the default focus ring color to white, instead of blue. Defaults to false.",
+        defaultValue: false,
+        type: {
+            required: true,
```

Comment:
Good question... in that case, we are marking them as required, but also assigning default values.

https://github.com/Khan/wonder-blocks/blob/fe1201745e7005d59764fe1dd8703db954f7e787/packages/wonder-blocks-clickable/src/components/clickable.js#L220-L225

Reply:
I do think it makes sense to give them default values, but I still think it's a bit confusing that they're marked as required, documentation-wise. In terms of architecture, I would expect those properties to be optional, personally. It okay though! I think that changing it would be out of scope here anyway.